packages/shared/src/components/modals/SquadMemberModal.tsx

```diff
@@ -91,6 +101,29 @@ export function SquadMemberModal({
             </SimpleTooltip>
           );
         }}
+        initialItem={
+          <button
+            type="button"
+            disabled={copying}
+            className="flex justify-start items-center py-3 px-6 hover:bg-theme-hover"
+            onClick={() => {
+              trackEvent({
+                event_name: AnalyticsEvent.ShareSquadInvitation,
+                extra: JSON.stringify({
+                  origin: Origin.SquadMembersList,
+                  squad: squad.id,
+                }),
+              });
```

Comment:
Ideally this should happen in the one hook accepting origin and squad id as props.
WDYT? (Might already be in @sshanzel hook?